site/layouts/partials/docs-versions.html

```diff
@@ -5,7 +5,7 @@
   <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="bd-versions">
     <li><a class="dropdown-item current" aria-current="true" href="/docs/{{ .Site.Params.docs_version }}/">Latest (5.0.x)</a></li>
     <li><hr class="dropdown-divider"></li>
-    <li><a class="dropdown-item" href="https://getbootstrap.com/docs/4.5/">v4.5.0</a></li>
+    <li><a class="dropdown-item" href="https://getbootstrap.com/docs/4.5/">v4.5.3</a></li>
```

Comment:
Should I change this to v4.5.x? It's not automated, but I might be able to automate it later.

Reply:
Might be better indeed